src/app/check_opam_switch/check_opam_switch.ml

```diff
@@ -0,0 +1,12 @@
+(**
+   Checks that the current switch contains all the packages specified
+   in the opam.export passed as an argument.
```

Comment:
I dream of an actual specified file format (with an extension like .json or .toml for syntax highlighting and formatting) that allows us to concisely and precisely list all dependencies and their versions. Perhaps an opam.export file could be generated from that file. Something that would look like a `Cargo.toml` but would also list the ocaml version.